docs/src/main/asciidoc/mongodb.adoc

```diff
@@ -588,7 +588,7 @@ This behavior can be disabled by setting the `quarkus.mongodb.health.enabled` pr
 
 == Metrics
 
-If you are using the `quarkus-smallrye-metrics` extension, `quarkus-mongodb` can provide metrics about the connection pools.
+If you are using the `quarkus-micrometer` or `quarkus-smallrye-metrics` extension, `quarkus-mongodb-client` can provide metrics about the connection pools.
 This behavior must first be enabled by setting the `quarkus.mongodb.metrics.enabled` property to `true` in your `application.properties`.
 
 So when you access the `/metrics` endpoint of your application you will have information about the connection pool status in the `vendor` scope.
```

Comment:
Need similar clarification here narrowing the vendor scope to the `quarkus-smallrye-metrics` extension.

Reply:
Thanks for catching that